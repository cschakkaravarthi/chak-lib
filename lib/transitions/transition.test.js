"use strict";
var __importDefault = (this && this.__importDefault) || function (mod) {
    return (mod && mod.__esModule) ? mod : { "default": mod };
};
exports.__esModule = true;
var react_1 = __importDefault(require("react"));
var testHelper_1 = require("../services/testHelper");
var menuSlide_1 = require("./menuSlide");
describe('MenuSlideTransition', function () {
    it('should render correctly', function () {
        var tree = testHelper_1.shallowRender(react_1["default"].createElement(menuSlide_1.MenuSlideTransition, { showToggle: true },
            react_1["default"].createElement("p", null, "Some content in the menu")));
        expect(tree).toMatchSnapshot();
    });
});
//# sourceMappingURL=transition.test.js.map