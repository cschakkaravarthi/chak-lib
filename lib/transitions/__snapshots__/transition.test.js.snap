// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MenuSlideTransition should render correctly 1`] = `
<CSSTransition
  classNames="slide-menu"
  in={true}
  timeout={300}
  unmountOnExit={true}
>
  <p>
    Some content in the menu
  </p>
</CSSTransition>
`;
